webpackjs/calcs.js

```diff
@@ -0,0 +1,346 @@
+    maxFeet: "100",
+    inputWidth: 0,
+    inputLinearFeet: 0,
+    output: 0,
+  },
+  beadsExtruded: {
+    type: "Width,LinearFeet",
+    heading: "Beads - for extruded machine",
+    label: "Pounds of beads",
+    maxWidth: "12",
+    maxFeet: "100",
+    inputWidth: 0,
+    inputLinearFeet: 0,
+    output: 0,
+  },
+  primer: {
+    type: "Width,LinearFeet",
+    heading: "Thermoplastic: Primer (sealant) - for extruded machine",
+    label: "Gallons of primer",
+    maxWidth: "12",
+    maxFeet: "10000",
+    inputWidth: 0,
+    inputLinearFeet: 0,
+    output: 0,
+  },
+  paintGallons: {
+    type: "Width,LinearFeet,Thickness",
+    heading: "Gallons Paint Used",
+    label: "Gallons needed",
+    maxWidth: "12",
+    maxFeet: "10000",
+    maxThickness: "15",
+    inputWidth: 0,
+    inputLinearFeet: 0,
+    inputThickness: 0,
+    output: 0,
+  },
+  beadsPaint: {
+    type: "Width,LinearFeet",
+    heading: "Beads (Pounds of beads)",
+    label: "Pounds of beads",
+    maxWidth: "12",
+    maxFeet: "10000",
+    inputWidth: 0,
+    inputLinearFeet: 0,
+    output: 0,
+  },
+  adhesive: {
+    type: "RPMS",
+    heading: '4" RPMS',
+    label: "Pounds of bituminous adhesive",
+    maxRPMS: "100",
+    inputRPMS: 0,
+    output: 0,
+  }
+}
+
+class Calcs extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      thermo60: objectProps.thermo60,
+      thermo90: objectProps.thermo90,
+      beadsExtruded: objectProps.beadsExtruded,
+      primer: objectProps.primer,
+      paintGallons: objectProps.paintGallons,
+      beadsPaint: objectProps.beadsPaint,
+      adhesive: objectProps.adhesive
+    };
+    this.thermo60GetOutputWidth = this.thermo60GetOutputWidth.bind(this);
+    this.thermo60getOutputLinearFeet = this.thermo60getOutputLinearFeet.bind(this);
+    this.thermo90GetOutputWidth = this.thermo90GetOutputWidth.bind(this);
+    this.thermo90getOutputLinearFeet = this.thermo90getOutputLinearFeet.bind(this);
+    this.beadsExtrudedGetOutputWidth = this.beadsExtrudedGetOutputWidth.bind(this);
+    this.beadsExtrudedGetOutputLinearFeet = this.beadsExtrudedGetOutputLinearFeet.bind(this);
+    this.primerGetOutputWidth = this.primerGetOutputWidth.bind(this);
+    this.primerGetOutputLinearFeet = this.primerGetOutputLinearFeet.bind(this);
+    this.paintGallonsGetOutputWidth = this.paintGallonsGetOutputWidth.bind(this);
+    this.paintGallonsGetOutputLinearFeet = this.paintGallonsGetOutputLinearFeet.bind(this);
+    this.paintGallonsGetOutputThickness = this.paintGallonsGetOutputThickness.bind(this);
+    this.beadsPaintGetOutputWidth = this.beadsPaintGetOutputWidth.bind(this);
+    this.beadsPaintGetOutputLinearFeet = this.beadsPaintGetOutputLinearFeet.bind(this);
+    this.adhesiveGetOutputRPMS = this.adhesiveGetOutputRPMS.bind(this);
+  }
+
+  thermo60GetOutputWidth() {
+    const inputWidth = event.target.value;
+    const output = inputWidth*(this.state.thermo60.inputLinearFeet/17.5609756097561);
```

Comment:
seeing some constants, could we pull these number values out? 

ex: 
```js
const thermo60Divisor = 17.5609756097561;
const thermo90Divisor = 12.1951219512195;
````

Reply:
Good call. I added a "divisor" field to each of the objectProps objects and used that for each respective function.